hoot-core/src/main/cpp/hoot/core/ops/ReplaceElementOp.cpp

```diff
@@ -39,6 +39,12 @@ namespace hoot
 
 HOOT_FACTORY_REGISTER(OsmMapOperation, ReplaceElementOp)
 
+ReplaceElementOp::ReplaceElementOp()
+  : _clearAndRemove(false)
+{
+}
+
+
```

Comment:
`_clearAndRemoved` wasn't initialized using the default constructor but was used later as in a conditional.